src/excited_workflow/source_datasets/land_cover.py

```diff
@@ -1,19 +1,117 @@
 """Ingest Copernicus land cover data."""
+from pathlib import Path
 from typing import Literal
 
+import numpy as np
 import xarray as xr
 import xarray_regrid  # noqa: F401
 
 from excited_workflow.source_datasets.protocol import DataSource
 from excited_workflow.source_datasets.protocol import get_freq_kw
 
 
+def _cftime_to_datetime(data: xr.DataArray) -> np.ndarray:
+    """Convert cftime dataarray values to a numpy datetime format.
+
+    Args:
+        data: DataArray containing the time values, e.g. ds["time"].
+
+    Returns:
+        Numpy array with a datetime64 dtype.
+    """
+    return np.array([np.datetime64(el) for el in data.to_numpy()])
+
+
+def dims_check(dat: xr.Dataset, target: xr.Dataset, var: str) -> bool:
```

Comment:
Looking at this again, the coordinates (and their values) are checked, right?
It's probably better to reflect that in the function definition and docstring
```suggestion
def coord_matches(dat: xr.Dataset, target: xr.Dataset, coord: str) -> bool:
```

Reply:
yep good point